__tests__/app.test.js

```diff
@@ -146,3 +146,28 @@ describe("/api/articles/:article_id", () => {
     });
   });
 });
+
+describe("/api/articles", () => {
+  describe("GET", () => {
+    test("200: responds with array of articles with comment counts in descending date order", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          const { articles } = response.body;
+          expect(articles).toBeSortedBy("created_at", { descending: true });
+          expect(articles[0]).toEqual({
+            author: "icellusedkars",
+            title: "Eight pug gifs that remind me of mitch",
+            article_id: 3,
+            topic: "mitch",
+            created_at: "2020-11-03T09:12:00.000Z",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            comment_count: "2",
+          });
+        });
+    });
+  });
```

Comment:
I would change this part of the test to iterate over the articles and assert the shape of each one. Technically your test doesn't prove that you have all the articles, even with the sorted by, because if you just have one article back then the sort will always be correct.

Consider using a forEach and checking each property has the keys you expect and the data type of the value that you expect. 

You should also have a length check to assert the correct number of articles are being returned